content/TrueCommand/TCGettingStarted/Install/Migrate.md

```diff
@@ -1,6 +1,13 @@
 ---
 title: "Migrate Legacy to 1.2+"
+description: "This article describes how to migrate Legacy versions of TrueCommand."
```

Comment:
legacy is not a proper noun and this is not specifically talking about a UI element so suggest
```suggestion
description: "This article describes how to migrate legacy versions of TrueCommand."
```

Reply:
Made suggested changes.